masterlist.yaml

```
@@ -12469,6 +12469,44 @@ plugins:
   - name: 'Falskaar.esm'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/2057/' ]
     msg:
+      - <<: *alsoUseX
+        subs: [ '[Falskaar - Bug Fixes](https://www.nexusmods.com/skyrimspecialedition/mods/19454/)' ]
+        condition: 'not active("Falskaar - Bug Fixes.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Better Dynamic Snow'
+          - '[Falskaar - Addons and Patches](https://www.nexusmods.com/skyrimspecialedition/mods/19454/)'
+        condition: 'active("Better Dynamic Snow.esp") and not active("Falskaar - Better Dynamic Snow Patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - Beyond Skyrim - Bruma'
```

@ddbb07 Besides this, it looks good to go.
```suggestion
          - 'Beyond Skyrim - Bruma'
```

---

Oh, how did i miss that, thanks for the suggestion